experimental/client-app/README.md

```diff
@@ -18,12 +18,61 @@ Within your terminal, open one tab to run the proxy and another to run the app.
 ```bash
 npm install
 npm build 
-npm run dev
+npm run start
 ```
 
-- `server` runs the proxy server to target api on port `3000`
-- `start` runs your app for development on port `8000`, reloading on file changes
-- `dev` uses npm-run-all to run the above `server` and `start` scripts in parallel (Perfect for cloudshell development.)
+To run locally:
+
+- `npm start` runs your app for development on port `8000`, reloading on file changes
+
+## Deploying Lit container
+
+```
+// Requires the following environment
+// variables to be set Cloud Run container
+REDIRECT_URI - uri pointing to auth api callback (i.e */auth/google)
+API_URL - url that points to server api to obtain cause/campaign information
+SITE_URL - url that points to current client app (this app)
+AUTH_API_URL - url that points to nodejs auth api url
+CLIENT_ID - oauth client id
+CLIENT_SECRET - oauth client secret
+```
+
+```bash
+export PROJECT_ID = your-project-id
+gcloud builds submit --tag gcr.io/$PROJECT_ID/lit-auth-website
+gcloud run deploy --image=gcr.io/$PROJECT_ID/lit-auth-website --port 8000
```

Comment:
Using Container Registry seems expedient during experimental R&D, but we're using Artifact Registry in practice.

I think the migration path is to use the existing website repository, but name it differently. E.g., 
us-central1-docker.pkg.dev/emblem-ci-ops/website/lit

The idea is that each "component" has a single AR instance and any services needed for that instance sit side-by-side by image name.

(Also okay for us to have a better name for "next-gen, Lit-based, Cymbal-enabled frontend" than `lit`)

Reply:
Would it be possible for me to add this as a separate task?